src/TaskBoard.js

```diff
@@ -45,10 +52,18 @@ export default function TaskBoard(){
 
     return (
         <div className="TaskBoard">
-          <CreateTaskForm onHandleSubmit = { handleFormSubmit } />
+          <CreateTaskForm 
+            onHandleSubmit = { handleFormSubmit }
+            onSetTaskInput = {setTaskInupt}
+            onSetAsignee = {setAsignee}
```

Comment:
@Tom-Sarfo you don't have to add the prefix `on` to every prop you pass when its a function. Usually the `on` prefix is left for handlers

Reply:
Thanks